content/en/docs/instrumentation/java/extensibility.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Extensibility
+description: Extensions add new features and capabilities to the agent without having to create a separate distribution or to fork this repository. For example, you can create custom samplers or span exporters, set new defaults, and embed it all in the agent to obtain a single jar file.
+---
+
+## Introduction
+
+Extensions are designed to override or customize the instrumentation provided by the upstream agent without having to create a new OpenTelemetry distribution or alter the agent code in any way.
```

Comment:
Nit: line-wrap paragraphs to 80 chars in this page.

Reply:
done